app/js/main.js

```diff
@@ -335,10 +356,7 @@ function deleteFrame(id) {
 
         updateFrameReel("delete", id);
         console.info(`There are now ${curFrame} captured frames`);
```

Comment:
With the condition gone, dedent all these statements one level (subject to further review).